src/ChannelServer/Network/ChannelPacketSender.cs

```diff
@@ -432,24 +432,24 @@ public static void ZC_ABILITY_LIST(Character character)
 			character.Connection.Send(packet);
 		}
 
-		/// <summary>
-		/// Broadcasts ZC_MOVE_SPEED in range of character, updating their move speed.
-		/// </summary>
-		/// <param name="character"></param>
-		public static void ZC_MOVE_SPEED(Character character)
+        /// <summary>
+        /// Broadcasts ZC_MOVE_SPEED in range of character, updating their move speed.
+        /// </summary>
+        /// <param name="entity"></param>
+        public static void ZC_MOVE_SPEED(IEntity entity)
 		{
 			var packet = new Packet(Op.ZC_MOVE_SPEED);
 
-			packet.PutInt(character.Handle);
-			packet.PutFloat(character.GetSpeed());
+			packet.PutInt(entity.Handle);
+			packet.PutFloat(entity.Speed);
 			packet.PutFloat(0);
 
 			// [i11257 (2016-03-25)]
 			{
 				packet.PutByte(0);
 			}
 
-			character.Map.Broadcast(packet, character);
+            entity.Map.Broadcast(packet, entity);
```

Comment:
Please set your IDE to use tabs, as outlined in our coding conventions, which can be found in the [readme](https://github.com/aura-project/melia#coding-conventions).